### DISCUSSION TITLE
No update on roadmap for over 2 months.

### DISCUSSION THREAD

**Discussion by product-is-king** [nammayatri/nammayatri#907] [Q&A] [answered]
Big fan of Namma yatri. However, not seeing any update to the roadmap for the last 2 months. 
Is it not being actively looked upon?

**Comment by Prithivirajkota** [comment, marked as answer]
@product-is-king Thank you for your interest in Namma Yatri. We have recently updated the roadmap. Please find the updated roadmap [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vRPdBJDIKyKHmAprGdLGfGb6Lt8j25xBHhV1Dr8f0kY8xhcP5DbBgAB5Fl-zvT4GjzNxwWzy81B1fDO/pubhtml?gid=685751497&single=true)

**Reply by product-is-king** [reply]
Great.. thank you.. god speed..

**Comment by privatejoel** [comment]
@Prithivirajkota  Following up on this, no update since the last update done on Jun 7th, would be awesome if you could update it.
